src/components/Hero.js

```diff
@@ -25,25 +27,103 @@ function Hero({handleLogIn}) {
           <MarketingButton primary onClick={handleLogIn}>
             Login
           </MarketingButton>
-          <a href="">
-            <MarketingButton>
-              <Octicon verticalAlign="middle" icon={getIconByName("book")} />
-              {" "}Read The Story
-            </MarketingButton>
-          </a>
+          <MarketingButton primary onClick={handleLogIn}>
+            Signup
+          </MarketingButton>
         </Wrapper>
       </SubHero>
       <TrustSection>
-        {/*
-        <small>Trusted By</small>
-        <ul>
-          <li>ONE</li>
-          <li>TWO</li>
-          <li>THREE</li>
-          <li>FOUR</li>
-        </ul>
-        */}
+        <Wrapper>
+          <ul>
+            <li>
+              <small>Trusted By</small>
+            </li>
+            <li>
+              <img src={"https://www.pinclipart.com/picdir/big/256-2564394_code-chrysalis-graphql-meetup-clipart.png"} />
```

Comment:
This and the other image should be local.

The test failure is because there is no `alt` text. 
```suggestion
              <img alt="graphql" src={"https://www.pinclipart.com/picdir/big/256-2564394_code-chrysalis-graphql-meetup-clipart.png"} />
```